github-orgs/kubeflow/org.yaml

```diff
@@ -240,6 +241,11 @@ orgs:
             - ScorpioCPH
             - ohmystack
             privacy: closed
+          Arm:
```

Comment:
We don't really use groups so unless you have a specific reason for adding a group please remove.

If you'd like Arm to be recognized as a contributor then please submit a PR adding it to
https://github.com/kubeflow/community/blob/master/member_organizations.yaml

Reply:
Hi @jlewi ! Thanks for this, I have removed the addition for the group. 

Thank you!